src/javascript/app/common/form_validation.js

```diff
@@ -92,6 +93,14 @@ const Validation = (() => {
                 });
             }
         }
+
+        // need to init Dropdown after we have responses from ws
+        const el_all_select = document.querySelectorAll('select');
+        el_all_select.forEach((el) => {
+            if (el.id !== 'tax_residence') {
```

Comment:
instead of specific id, can we check for the attribute 'multiple'? e.g. something like `if (el.getAttribute('multiple') === 'multiple')` since I assume we want to leave any multi-selects alone, and that is a better way of catching all of them instead of individually excluding them